crates/ln-dlc-node/src/node.rs

```diff
@@ -481,6 +490,89 @@ impl Node {
             }
         });
 
+        let dlc_man = self.dlc_manager.clone();
+        let pending_trades = self.pending_trades.clone();
+        let sub_channel_manager = self.sub_channel_manager.clone();
+        let dlc_message_handler = self.dlc_message_handler.clone();
+        let peer_manager = self.peer_manager.clone();
+
+        tokio::spawn({
+            let dlc_man = dlc_man.clone();
+            let sub_channel_manager = sub_channel_manager.clone();
+            let dlc_message_handler = dlc_message_handler.clone();
+            let peer_manager = peer_manager.clone();
+
+            async move {
+                loop {
+                    Node::process_incoming_messages_internal(
+                        &dlc_message_handler,
+                        &dlc_man,
+                        &sub_channel_manager,
+                        &peer_manager,
+                    )
+                    .unwrap();
+                    tokio::time::sleep(Duration::from_secs(30)).await;
```

Comment:
We should add a todo here that this sleep is arbitrary and me might need to change it.